slack bot should extend the accepted semantic release version to 21.0

when I build my project with the latest semantic-release, I get:

```
npm ERR! While resolving: semantic-release-slack-bot@4.0.0
npm ERR! Found: semantic-release@21.0.0
npm ERR! node_modules/semantic-release
npm ERR!   dev semantic-release@"21.0.0" from the root project
npm ERR!   peer semantic-release@">=18.0.0" from @semantic-release/changelog@6.0.3
npm ERR!   node_modules/@semantic-release/changelog
npm ERR!     dev @semantic-release/changelog@"6.0.3" from the root project
npm ERR!   6 more (@semantic-release/commit-analyzer, ...)
```

see https://github.com/semantic-release/semantic-release/releases/tag/v21.0.0

--- a/package.json
+++ b/package.json
@@ -69,7 +69,7 @@
     "sinon": "^9.2.1"
   },
   "peerDependencies": {
-    "semantic-release": ">=11.0.0 <22.0.0"
+    "semantic-release": ">=11.0.0"
   },
   "config": {
     "commitizen": {

--- a/package.json
+++ b/package.json
@@ -69,7 +69,7 @@
     "sinon": "^9.2.1"
   },
   "peerDependencies": {
-    "semantic-release": ">=11.0.0 <21.0.0"
+    "semantic-release": ">=11.0.0 <22.0.0"
   },
   "config": {
     "commitizen": {

